src/style.scss

```diff
@@ -0,0 +1,186 @@
+/* #region root */
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
```

Comment:
you can also keep the variables in separate file to share them across different BEM block, which also should be files, imported in index.scss file